packages/contracts/interfaces/IAnonAadhaarVote.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+interface IAnonAadhaarVote {
+
+    struct Proposal {
+        string description;
+        uint256 voteCount;
+    }
+
+    event Voted(address indexed _from, uint256 indexed _propositionIndex);
+
+    function voteForProposal(uint256 proposalIndex, uint256[2] calldata _pA, uint[2][2] calldata _pB, uint[2] calldata _pC, uint[5] calldata _publicInputs, uint256 signal) external;
+
+    function getProposalCount() external returns (uint256);
+
+    function getTotalVotes() external returns (uint256);
```

Comment:
Is there an advantage on having an inerface to derive functions? We can keep the structs and event only and import them in the implementation contract (instead of that contract deriving this interface). I think interface like this is relevant when we expect to have other implementation of the all signature. wdyt?